tools/clang/test/CodeGenSPIRV/early-return-float4.hlsl

```diff
@@ -0,0 +1,94 @@
+// Run: %dxc -T ps_6_0 -E main
+
+// CHECK: [[v4f1:%\d+]] = OpConstantComposite %v4float %float_1 %float_1 %float_1 %float_1
+// CHECK: [[v4f2:%\d+]] = OpConstantComposite %v4float %float_2 %float_2 %float_2 %float_2
+// CHECK: [[v4f3:%\d+]] = OpConstantComposite %v4float %float_3 %float_3 %float_3 %float_3
+// CHECK: [[v4f4:%\d+]] = OpConstantComposite %v4float %float_4 %float_4 %float_4 %float_4
+// CHECK: [[v4f5:%\d+]] = OpConstantComposite %v4float %float_5 %float_5 %float_5 %float_5
+// CHECK: [[v4f6:%\d+]] = OpConstantComposite %v4float %float_6 %float_6 %float_6 %float_6
+// CHECK: [[v4f7:%\d+]] = OpConstantComposite %v4float %float_7 %float_7 %float_7 %float_7
+// CHECK: [[v4f8:%\d+]] = OpConstantComposite %v4float %float_8 %float_8 %float_8 %float_8
+// CHECK: [[v4f9:%\d+]] = OpConstantComposite %v4float %float_9 %float_9 %float_9 %float_9
+
+// CHECK: [[out:%\d+]] = OpVariable %_ptr_Output_v4float Output
+
+float4 main(float4 input: COLOR): SV_TARGET {
+  int a, b;
+  bool cond = true;
+
+  while(cond) {
+    switch(b) {
+// CHECK: %switch_1 = OpLabel
+      case 1:
+        a = 1;
+// CHECK:      OpStore [[out]] [[v4f1]]
+// CHECK-NEXT: OpReturn
+        return float4(1.0, 1.0, 1.0, 1.0);
```

Comment:
Ah, sorry. I'll change how stage IO variables are handled. So these tests will be killed very soon. I think we should just have test cases for normal functions that got called by `main`.